src/index.html

```diff
@@ -11,7 +11,162 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset>
+        <legend>Personal Information</legend>
+
+        <label class="pers-inf">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          required
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="pers-inf">
+          Name:
+          <input
+          type="text"
+          name="name"
+          required
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="pers-inf">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="pers-inf">
+          Full date of birth:
+          <input type="date"
+          name="bithday"
+          >
+        </label>
+
+        <label class="pers-inf">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="accept"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Regestration</legend>
+
+        <label class="pers-inf">
+         E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="pers-inf">
+          Password
+          <input
+          type="password"
+          name="password"
+          minlength="5"
+          maxlength="32"
+          >
+        </label>
+
+      </fieldset>
+
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <label class="pers-inf">
```

Comment:
move label to inputs
it chooses the first option when I click on `Do you love cats?` now

Reply:
<img width="234" alt="image" src="https://user-images.githubusercontent.com/102411130/225856895-48eeb9eb-72ad-4abd-9596-6a5c3a0f62c5.png">